content/docs/deploy/k8s/ingress.md

```diff
@@ -351,6 +351,37 @@ ingress.pomerium.io/service_proxy_upstream: 'true'
 
 Unless you disabled direct traffic to Endpoints, Pomerium would load balance the requests to the upstream endpoints. See the [Load Balancing](/docs/capabilities/load-balancing) guide for details, and use relevant Ingress annotations to fine tune load balancing and health checks.
 
+```yaml
+ingress.pomerium.io/lb_policy: 'lb_policy_option'
+```
+
+The `lb_policy` has the following options:
+
+| **Load Balancer Policy options** |
+| :-- |
+| [`ROUND_ROBIN`](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/load_balancers#weighted-round-robin) |
```

Comment:
please add a comment that each pod has equal weight, and that cannot be customized.